src/Enum/OrientationEnum.php

```diff
@@ -17,6 +17,10 @@ class OrientationEnum
 
     public static function getOrientations(): array
     {
-        return self::ORIENTATIONS;
+        return [
+            self::ORIENTATION_VERTICAL => 'global.generate_label.orientation.vertical',
```

Comment:
This should be corrected in `self::ORIENTATIONS` instead? Or remove `self::ORIENTATIONS`, because it's used nowhere?

Please consider